src/include/splash/ParallelDataCollector.hpp

```diff
@@ -312,6 +312,12 @@ namespace splash
                 Dimensions &sizeRead,
                 void* buf) throw (DCException);
 
+        CollectionType* readMeta(int32_t id,
```

Comment:
not sure if relevant, but `read` is not documented either at this place but just in the `DataCollector` interface (?)